utils/dataloaders.py

```diff
@@ -52,53 +55,69 @@ def get_fashion_mnist_dataloaders(batch_size=128,
     return train_loader, test_loader
 
 
-def get_dsprites_dataloader(batch_size=128,
-                            path_to_data=os.path.join(DIR, '../data/datadsprites/dsprites_data.npz')):
+def get_dsprites_dataloader(batch_size=128):
     """DSprites dataloader."""
-    dsprites_data = DSpritesDataset(path_to_data,
-                                    transform=transforms.ToTensor())
-    dsprites_loader = DataLoader(dsprites_data, batch_size=batch_size,
-                                 shuffle=True)
-    return dsprites_loader
-
-
-def get_chairs_dataloader(batch_size=128,
-                          path_to_data=os.path.join(DIR, '../data/chairs/rendered_chairs_64')):
+    root = os.path.join(DIR, '01_data', 'dsprites-dataset')
```

Comment:
why having an intermediate 01_data folder for all of them ?? doesn't seem useful and is not very clean